### PR TITLE
SG-22646 Replaced references to Shotgun with ShotGrid

### PR DESCRIPTION
**Pull request by eshokrgozar**

### CODE REVIEW

**Review by jfboismenu — APPROVED** [review]
👍 I'm assuming the process to recompile the plugin was well documented then? I don't see any changes to the docs.

**Comment by jfboismenu** [comment]
@mathurf , your turn!

**Comment by eshokrgozar** [comment]
> 👍 I'm assuming the process to recompile the plugin was well documented then? I don't see any changes to the docs.

Yeah. So the README has a bunch of information about this. The part that was missing was in [this PR](https://github.com/shotgunsoftware/tk-framework-adobe/pull/18). I'm not sure if we need to fix the timecode thing or just disable it as mentioned in the PR. But it doesn't work with it on.

**Review by mathurf — CHANGES_REQUESTED** [review]
found this Shotgun mention
image

**Comment by eshokrgozar** [comment]
> found this Shotgun mention
> image

fixed

**Review by mathurf — APPROVED** [review]
LGTM

File: com.sg.basic.adobe.version [hunk 1]
```diff
@@ -1 +1 @@
-v1.1.1
+v1.1.3
```
[the remark this answers is not in the document]
Comment (mathurf) [reply com.sg.basic.adobe.version:1]: JF told me v1.1.3 is fine

**Pull request merged**